speechbrain/core.py

```diff
@@ -269,6 +269,13 @@ def parse_arguments(arg_list=None):
         type=int,
         help="Number of optimizer steps to run. If not passed, all epochs are run.",
     )
+    parser.add_argument(
+        "--white_tqdm",
```

Comment:
why white? not just "default"

Reply:
tqdm_default_color?

if we can specify between text & bg color, maybe we need to disinguish that too